include/envoy/ssl/connection.h

```diff
@@ -42,6 +42,12 @@ class Connection {
    *         certificate, or no SAN field, or no URI.
    **/
   virtual std::string uriSanPeerCertificate() PURE;
+
+  /**
+   * @return the base64 encoded representation of the peer certificate. Returns "" if there is no
```

Comment:
Envoy style nit, return type goes in return comment: `@return std::string the base64 encoded...`